_dev/pm2/services-debug.json

```diff
@@ -165,19 +112,20 @@
       "min_uptime": "2m"
     },
     {
-      "name": "payments server PORT 3031",
+      "name": "payments-debug",
```

Comment:
Having inconsistent names when running in debug mode vs. not means that the VSCode scripts are going to have difficulty, for example the auth-server VSCode task runs a command like so:
``` 
     "command": "pm2 stop 'auth-server key server PORT 9000'"
```

Reply:
pm2 gets confused when you have two different `apps` with the same name. maybe we just ditch the non-debug version